easypost/report.py

```diff
@@ -8,7 +8,20 @@ class Report(AllResource, CreateResource):
     def create(cls, api_key=None, **params):
         """Create a report."""
         requestor = Requestor(api_key)
-        url = "%s/%s" % (cls.class_url(), params["type"])
+        url = "%s/%s" % (cls.class_url(), params["type"]) + "?"
```

Comment:
Please us an f string here, it's much easier to read than mentally trying to slot in variables to a placeholder.